pandas/core/indexes/accessors.py

```diff
@@ -324,8 +321,7 @@ def __new__(cls, data):
 
         orig = data if is_categorical_dtype(data) else None
         if orig is not None:
-            data = Series(orig.values.categories, name=orig.name, copy=False)
-
+            data = Series(orig.values.to_list(), name=orig.name, copy=False)
```

Comment:
so we don't want to convert to a list, I think rather we should still operate on the categories
and just return a new Categorical

Reply:
I'm not sure I understand - is `data` supposed to be a `pandas.Categorical`? If so, then it won't pass the 
```
        if not isinstance(data, ABCSeries):
            raise TypeError(
                "cannot convert an object of type {0} to a "
                "datetimelike index".format(type(data))
            )
```
test when we get to
```
        if is_datetime64_dtype(data.dtype):
            return DatetimeProperties(data, orig)
```

Reply:
>  is data supposed to be a pandas.Categorical?

At this point `data` is a Series with CategoricalDtype.

I think @jreback's point is that we don't want to call `to_list()` because that is costly